Hardware/Hardware/CardReader/CardPresentTransactionDetails.swift

```diff
@@ -28,14 +28,24 @@ public struct CardPresentTransactionDetails: Codable, Equatable, GeneratedFakeab
     /// (Only applicable to EMV payments) The authorization data from the card issuer.
     public let emvAuthData: String?
 
+    /// If this payment is from a card wallet, this contains the details of the card wallet.
+    public let wallet: Wallet?
+
+    /// Identifies which network this charge was processed on.
+    /// Contains SCPCardBrand represented as a nullable NSNumber.
+    /// Only available after confirming the PaymentIntent.
+    public let networkDouble: Double?
```

Comment:
What do you think if we rename this `networkDouble` for clarity? Something like `cardBrandAssociatedIdentifier` or something similar that refers to the card brand associated value as an ID? We can also add a link to https://stripe.dev/stripe-terminal-ios/docs/Enums/SCPCardBrand.html in the comment for future reference.